base/abstractarray.jl

```diff
@@ -1254,8 +1254,11 @@ end
 """
     getindex(A, inds...)
 
-Return a subset of array `A` as specified by `inds`, where each `ind` may be,
-for example, an `Int`, an [`AbstractRange`](@ref), or a [`Vector`](@ref).
+Return a subset of array `A` as selected by the indices `inds`.
+
+Each index may be any [supported index type](@ref man-supported-index-types), such
+as an [`Integer`](@ref), [`CartesianIndex`](@ref), [range](ref `range`), or [array](@ref man-multi-dim-arrays) of supported indices.
+A [:](ref `Colon`) can be used to select all elements in some dimension, and a [`BitArray`](@ref) can be used to select all `true` elements.
```

Comment:
```suggestion
as an [`Integer`](@ref), [`CartesianIndex`](@ref), [range](@ref AbstractRange), or [array](@ref man-multi-dim-arrays) of supported indices.
A [:](@ref Colon) can be used to select all elements in some dimension, and a [`BitArray`](@ref) can be used to select all `true` elements.
```

I _think_ these are the correct `@ref` syntaxes.